app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Users
+  class SessionsController < Devise::SessionsController
+    # before_action :configure_sign_in_params, only: [:create]
+
+    # GET /resource/sign_in
+    # def new
+    #   super
+    # end
+
+    # POST /resource/sign_in
+    def create
+      puts 'TEA'
+      user = find_user
+
+      unless user
+        wrong_result and return
+      end
+
+      if user&.valid_password?(params[:password])
+        set_flash_message :notice, :signed_in
+        sign_in(user)
+        redirect_to user_path(user)
+      else
+        wrong_result
+      end
+      # super
+    end
+
+    # DELETE /resource/sign_out
+    # def destroy
+    #   super
+    # end
+
+    protected
+
+    def configure_sign_in_params
+      devise_parameter_sanitizer.permit(:sign_in, keys: [:identificator])
```

Comment:
The right word in English for this is `identifier`
```suggestion
      devise_parameter_sanitizer.permit(:sign_in, keys: [:identifier])
```

Reply:
Done! :$